openflow_input/bsn_tlv

```diff
@@ -888,3 +888,15 @@ struct of_bsn_tlv_push_vlan_on_ingress : of_bsn_tlv {
     uint16_t type == 128;
     uint16_t length;
 };
+
+struct of_bsn_tlv_apply_packets : of_bsn_tlv {
```

Comment:
Since it `apply packets` isn't super-obvious as a name (or concept), maybe it would be good to add a short comment with the generic purpose (no internal info - public repo). 

I realize this applies to other TLVs in this file as well..